packages/jsii-pacmak/package.json

```diff
@@ -64,6 +64,7 @@
     "@types/yargs": "^16.0.0",
     "eslint": "^7.21.0",
     "jest": "^26.6.3",
+    "jsii": "^0.0.0",
```

Comment:
Intentional? pacmak depends on jsii?

Not familiar with the jsii layering but smells fishy.

Reply:
New tests actually use jsii to compile some tiny projects to validate code-gen output. This is very practical.